packages/core/src/charts/donut/donut.spec.js

```diff
@@ -666,8 +666,8 @@ donutDataSets.forEach((datasetName) => {
 
                     donutChart.on('customClick', callback);
                     firstSlice.dispatch('click');
-                    expect(callback.mock.calls.length).toBe(1);
-                    expect(callback.mock.calls[0].length).toBe(3);
+                    expect(callback.mock.calls).toHaveLength(1);
```

Comment:
I will probably review this settings, as I do prefer not to use matchers as much as possible, relying on regular JS and a .toEqual or .toBe for everything.

Reply:
Some of these changes came from jest recommended rules. I can revert. But it is the recommended way

Reply:
No prob, I will polish it later on. Not a big deal.